src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.String.cs

```diff
@@ -122,7 +122,7 @@ private void WriteStringEscapeProperty(ReadOnlySpan<char> propertyName, int firs
                     // Cannot create a span directly since it gets assigned to parameter and passed down.
                     unsafe
                     {
-                        char* ptr = stackalloc char[length];
+                        char* ptr = stackalloc char[JsonConstants.StackallocThreshold];
```

Comment:
These ones using a pointer, is it safe to do like this?

Reply:
Yup! But change the line below to read `new Span<char>(ptr, const)` instead of `(..., length)` so that everything matches.

Reply:
Updated